plugins/_getdir/_getdir.css

```diff
@@ -1 +1,11 @@
+html {height: 100%;}
+body {background-color: window; color: windowtext; border: 0px; margin: 0px; padding: 0px; -moz-user-select:none; overflow: hidden; height: 100%;}
+.dir-list {display: flex; flex-direction: column; align-items: stretch; height: 100%;}
+.dir-list div { padding-top: 1px; padding-bottom: 1px; padding-left: 0px; padding-right: 0px; cursor:default; font-size: 11px; font-family: Tahoma, Arial, Helvetica, sans-serif; }
+.search-bar {display: flex; flex-direction: row;}
+.search-bar input[type="text"] {flex-grow: 1;}
+.rmenuobj {margin-bottom: 5px; border-width: 0; overflow-y: auto;}
+.rmenuitem { color: windowtext; }
+.rmenuitemselected { color: highlighttext; background-color: highlight; }
+
```

Comment:
Please resolve the text overflow of menu items in the directory list. We do not want to create a new line.

Reply:
Good catch! The directory name wraps itself when it's longer than the width of the directory list. 
![Screenshot 2024-06-24 110813](https://github.com/Novik/ruTorrent/assets/26022962/b5574e46-5dfb-4bcb-8fcb-96692799889a)

I think a single `white-space: nowrap;` rule will do the magic. A horizontal scrollbar appears, if users want to see the full long long directory name.
![Screenshot 2024-06-24 111248](https://github.com/Novik/ruTorrent/assets/26022962/fb6e153d-becb-450a-b47c-7fe4f7eb70d5)